src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs

```diff
@@ -67,10 +68,22 @@ public static IDisposable Push([NotNull] ElmScope scope, [NotNull] ElmStore stor
             Current = scope;
             Current.Parent = temp;
 
+            // Get the scope message
+            string scopeMessage = null;
+            var scopeLogValues = Current._state as ILogValues;
+            if(scopeLogValues != null)
+            {
+                scopeMessage = scopeLogValues.Format();
```

Comment:
@loudej @lodejard : Should I get rid of `.Format()` all together? I ask this because as part of the rename of `BeginScope` to `BeginScopeImpl`, I am coming across code (across repos) where `ToString()`  is being invoked on the state. Depending on what we decide here it would impact the number of changes that I will have to make across the repos to reflect the renaming change.